test/screenshot/README.md

```diff
@@ -1,22 +1,144 @@
 # Screenshot Tests
 
-## Local dev server
+Prevent visual regressions by running screenshot tests on every PR.
 
-Run:
+## Quick start
+
+Add the following to your `~/.bash_profile` (Mac) or `~/.bashrc` (Linux):
+
+```bash
+export CBT_USERNAME='you@example.com'
+export CBT_AUTHKEY='example'
+export MDC_GCLOUD_SERVICE_ACCOUNT_KEY_FILE_PATH='path/to/gcp-credentials.json'
+```
+
+In a new terminal, run:
+
+```bash
+npm run screenshot:test
+```
+
+This will take several minutes to complete.
+
+When the tests finish, you should see a link to the diff report page in the terminal:
+
+```
+Found 0 screenshot diffs!
+https://storage.googleapis.com/mdc-web-screenshot-tests/advorak/2018/05/22/17_49_05_673/d3a549bb4/report.html
+```
+
+Next, modify a Sass file, and rerun the tests:
+
+```bash
+echo '.mdc-button:not(:disabled){color:red}' >> packages/mdc-button/mdc-button.scss
+npm run screenshot:test
+```
+
+You should see something like this in the terminal:
+
+```
+Found 65 screenshot diffs!
+https://storage.googleapis.com/mdc-web-screenshot-tests/advorak/2018/05/22/16_23_58_618/af3d7271f/report.html
+```
+
+## Basic usage
+
+### Updating "golden" screenshots
+
+```bash
+npm run screenshot:update-goldens
+```
+
+In addition to running the tests, this command will also update your local `test/screenshot/golden.json` file with the 
+newly captured screenshots.
+
+### Rerunning flaky tests
+
+If a single screenshot fails to render, you can rerun it without running the entire suite:
+
+```bash
+npm run screenshot:update-goldens -- \
+  --mdc-include-browser=ie@11 \
+  --mdc-include-url=mdc-button/classes/dense
+```
+
+**IMPORTANT:** Note the `--` between `screenshot:test` and `--help`. This is required by `npm`.
```

Comment:
The example above uses `update-goldens` but the strings here assume `screenshot:test -- --help`. These should be consistent.

Reply:
All of the `npm run screenshot:*` commands accept the same CLI flags.

How can I make that clearer?

Reply:
Add another note under advanced CLI usage to say the flags work for both test and update-goldens.

(I'm assuming most of the options aren't _relevant_ for the other commands, like build and upload...)

Reply:
Done